en/docs/references/performance-analysis.md

```diff
@@ -1,18 +1,77 @@
+Traditionally, performance analysis includes performance debugging which involves measuring the performance of services by load testing them. The Performance Analyzer allows you to measure performance at the application development stage, significantly reducing the time and cost spent on performance debugging.
+
+A component may have more than one execution path serving requests. The Performance Analyzer tool can analyze all the execution paths and determine the **performance-critical path**: the path that is estimated to take the longest time to process a single request.
+
+## Advantages of using the Performance Analyzer
+
+Making accurate performance estimations before releasing an application you have developed is important due to the following reasons:
+
+- You can identify the performance-critical path of a component at development time. 
+- The forecasts you obtain for individual connector actions help identify any bottlenecks in each execution path.
+- You can detect any performance anti-patterns and low-quality code in your component implementation that impacts the overall performance. 
+- When you create service-level agreements (SLAs) for your application, an accurate performance forecast allows you to provide valuable insights relating to the scalability of the application and the ways to optimize its performance.
+- You can avoid missing significant performance characteristics when combining multiple applications, including external applications.
 - You can correctly assess the cost and feasibility of running the application in a production environment.
+- You can reduce the time and cost spent on debugging performance issues by eliminating or minimizing iterative load testing efforts.
+
+## Use the Performance Analyzer
+
+You can use the Choreo Performance Analyzer on the Web Editor in low-code mode. 
+
+### Analyze the performance-critical path in real-time
+
+To view the real-time analysis of the performance-critical path of your component, follow the steps below:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev). 
+2. Navigate to your component and click **Overview** from the left pane. 
+3. Click the **Edit Code** to access the Web Editor.
+
+#### Overall performance of the performance-critical path
+
+On the low-code editor, you will see a bar with the summarized overall performance estimates for each resource function. By default, this banner displays the forecasted latency and throughput for a selected range of user count for the performance-critical path. The upper bound of the user count range is set based on the availability of historical data. Since the Performance Analyzer tool derives these metrics in real-time,  any changes to the component will update the metrics in real-time. 
+
+![Real-time performance analysis of the critical path](../assets/img/references/performance-analysis/real-time-performance-analysis.png){.cInlineImage-threeQuarter}
```

Comment:
`{.cInlineImage-threeQuarter}` is not complied properly in the preview for each figure

Reply:
Yes. GitHub doesn't render CSS. It will be applied in the live version.